.storybook/stories/DataFiltering/TableFilteringMedium.test.tsx

```diff
@@ -1,23 +1,43 @@
 import '@testing-library/jest-dom';
 import 'html-validate/jest';
 import { axe, toHaveNoViolations } from 'jest-axe';
-import { cleanup, render, waitFor, act } from '../../../test-utils';
+import { cleanup, render } from '../../../test-utils';
 import { TableFilteringMedium } from './TableFilteringMedium';
 import { useSortAndFilter } from './lib/useSortAndFilter';
-import { useData } from './lib/utils';
 import { DataProvider, SortAndFilterProvider } from './lib/contexts';
+import { generateBusinessData } from './lib/generateBusinessData';
 
 expect.extend(toHaveNoViolations);
 
 afterEach(cleanup);
 
-function TableFilteringMediumTest({ loading }: { loading: boolean }) {
+const generatedData = generateBusinessData()
+	.slice(0, 10)
+	.map((i, index) => ({ ...i, index }));
+
+function TableFilteringMediumLoaded() {
 	const sortAndFilter = useSortAndFilter();
-	const { filters, pagination, sort } = sortAndFilter;
-	const data = useData({ filters, pagination, sort });
+	const data = {
+		loading: false,
+		data: generatedData,
+		totalPages: 10,
+		totalItems: 100,
```

Comment:
noting that only 10 items will actually exist in the array but this is probably ok for now